### ISSUE TITLE
Calibration test data in two subdirectories Cat-A and LevelA

### ISSUE THREAD

**Issue by morcuended** [cta-observatory/cta-lstchain#1108]
We currently use calibration test files under two subdirectories of monitoring/PixelCalibration Cat-A (naming we use for data processing) and LevelA (deprecated). We should only use the `Cat-A` directory to avoid confusion.

**Comment by morcuended** [comment]
No, this has nothing to do with lstosa. It's in the lstchain tests for calibration tools. It's the way they are stored in the test data directory structure and hence used in the lstchain tests

**Comment by rlopezcoto** [comment]
Ah ok, thanks, so is this already removed in #1054 @maxnoe
